Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -24,7 +24,11 @@ public class ArcGISARView: UIView {
     public let arSCNView = ARSCNView(frame: .zero)
     
     /// The initial transformation used for a table top experience.  Defaults to the Identity Matrix.
-    public private(set) var initialTransformation = AGSTransformationMatrix.identity
```

Comment:
This is the natural way of doing this sort of thing. I suspect you moved away from it because of this?

```swift
public func setInitialTransformation(initialTransformation: AGSTransformationMatrix) -> Bool {
    _initialTransformation = initialTransformation
    return true
}
```

If so, the solution is to add `self`:

```swift
self.initialTransformation = initialTransformation
```

Reply:
The `setInitialTransformation(...)` methods were interfering with the `initialTransformation` property when I had it the original way (`public private(set) var...`).  So I made the property a get-only and have a backing private variable.

Reply:
I'm not sure the `setInitialTransformation` methods themselves were the cause of the problem. I was able to build using the old `initialTransformation` property by updating all of the references from `_initialTransformation` to `initialTransformation`, with the exception of the above mentioned method needing to disambiguate `initialTransformation` (local variable) from `initialTransformation` (instance variable:

```swift
public func setInitialTransformation(_ initialTransformation: AGSTransformationMatrix) -> Bool {
    self.initialTransformation = initialTransformation
    return true
}
```

What you have is functionally equivalent, so it's fine. I just wanted you to know that what you had before should work (with a slight tweak).